plugin/src/test/java/org/opensearch/ml/utils/MLNodeUtilsTests.java

```diff
@@ -66,6 +66,149 @@ public void testValidateSchema() throws IOException {
         MLNodeUtils.validateSchema(schema, json);
     }
 
+    @Test
+    public void testValidateEmbeddingInputWithGeneralEmbeddingRemoteSchema() throws IOException {
+        String schema = "{\n"
+            + "    \"type\": \"object\",\n"
+            + "    \"properties\": {\n"
+            + "        \"parameters\": {\n"
+            + "            \"type\": \"object\",\n"
+            + "            \"properties\": {\n"
+            + "                \"texts\": {\n"
+            + "                    \"type\": \"array\",\n"
+            + "                    \"items\": {\n"
+            + "                        \"type\": \"string\"\n"
+            + "                    }\n"
+            + "                }\n"
+            + "            },\n"
+            + "            \"required\": [\n"
+            + "                \"texts\"\n"
+            + "            ]\n"
+            + "        }\n"
+            + "    }\n"
+            + "}";
+        String json = "{\"text_docs\":[ \"today is sunny\", \"today is sunny\"]}";
+        MLNodeUtils.validateSchema(schema, json);
+    }
+
+    @Test
+    public void testValidateRemoteInputWithGeneralEmbeddingRemoteSchema() throws IOException {
+        String schema = "{\n"
+            + "    \"type\": \"object\",\n"
+            + "    \"properties\": {\n"
+            + "        \"parameters\": {\n"
+            + "            \"type\": \"object\",\n"
+            + "            \"properties\": {\n"
+            + "                \"texts\": {\n"
+            + "                    \"type\": \"array\",\n"
+            + "                    \"items\": {\n"
+            + "                        \"type\": \"string\"\n"
+            + "                    }\n"
+            + "                }\n"
+            + "            },\n"
+            + "            \"required\": [\n"
+            + "                \"texts\"\n"
+            + "            ]\n"
+            + "        }\n"
+            + "    }\n"
+            + "}";
+        String json = "{\"parameters\": {\"texts\": [\"Hello\",\"world\"]}}";
+        MLNodeUtils.validateSchema(schema, json);
+    }
+
+    @Test
+    public void testValidateEmbeddingInputWithTitanTextRemoteSchema() throws IOException {
+        String schema = "{\n"
```

Comment:
Why not just use `TITAN_TEXT_EMBEDDING_MODEL_INTERFACE_INPUT`? 
So in future if someone changed `TITAN_TEXT_EMBEDDING_MODEL_INTERFACE_INPUT`, we can catch the error with this IT. 
If we define a new variable here and someone changed `TITAN_TEXT_EMBEDDING_MODEL_INTERFACE_INPUT` by mistake, how can we ensure this UT can catch that error ?

Reply:
Changed